native_client/python/client.py

```diff
@@ -64,13 +86,13 @@ def main():
         print('Loaded language model in %0.3fs.' % (lm_load_end), file=sys.stderr)
 
     fs, audio = wav.read(args.audio)
-    # We can assume 16kHz
+    if fs != 16000:
+        audio = convert_samplerate(args.audio)
```

Comment:
nit: we should `fs = 16000` after calling `convert_samplerate()` and then update all uses of hard-coded `16000` to use `fs`